pkg/ebpf/ksyms.go

```diff
@@ -63,8 +93,13 @@ func (ec *existCache) verifyKernelFuncs(requiredKernelFuncs []string) (map[strin
 			fields := bytes.Fields(scanner.Bytes())
 			if len(fields) >= 3 {
 				if idx := check.Search(fields[2]); idx >= 0 {
+					s, err := strconv.ParseUint(string(fields[0]), 16, 64)
+					if err != nil {
+						return nil, fmt.Errorf("failed to parse kallsyms address for symbol %s: %w", string(fields[2]), err)
```

Comment:
do we want to fail here the whole process in case one symbol had failed to parse?